### DISCUSSION TITLE
Azuracast set  new domain problem

### DISCUSSION THREAD

**Discussion by megasmorf** [AzuraCast/AzuraCast#6021] [Q&A]
### Installation Method

Docker Installation

### AzuraCast Release Channel

Rolling Release Channel

### Current AzuraCast Version

 Rolling Release #03b593f (2022-12-27 14:08) • Docker • PHP 8.1

### What happened?

Hello sorry for my issue but i not find my problem in help topics 

I have installed with docker on a host server where I already have a cloud app and a domain name.
Azyracast runs in docker under the central address of the central server (for example 1.2.3.4:port/login) what I unfortunately have not understood is how I will be able to put another domain that I have in the docker installation or even better it should be in subdomain.
Example when I set up the cloud app I made the necessary settings in the apache config in the sites enable folder and everything is fine now with the radio app installed in docker all this how do I actually do it I looked in the documentation and couldn't figure it out

**Comment by sadoj-kuro** [comment]
It's the same thing for me, I changed my domain and since then I can't access the site with the domain, whereas it works fine with the IP. I've correctly reconfigured the DNS, I've defined the domain in the site's configurations by accessing it via the IP, but nothing works. I wonder if there might be a place where the value of the old domain is stored in a file or elsewhere.
